en: update target settings in config file (#453)
pingcap/docs-dm: en/task-configuration-file-full.md

  host: "192.168.0.1"
  port: 4000
  user: "root"
  password: "/Q7B9DizNLLTTfiZHv9WoEAKamfpIUs="  # It is recommended to use a password encrypted with dmctl.
  max-allowed-packet: 67108864                  # Sets the "max_allowed_packet" limit of the TiDB client (that is, the limit of the maximum accepted packet) when DM internally connects to the TiDB server. The unit is bytes. (67108864 by default)

>                                                 # Since DM v2.0.0, this configuration item is disabled, and DM will automatically obtain the "max_allowed_packet" to connect to TiDB.

Reviewer: ```suggestion
                                                # Since DM v2.0.0, this configuration item is deprecated, and DM automatically obtains the "max_allowed_packet" value to connect to TiDB.
```

>                                                 # Since DM v2.0.0, this configuration item is deprecated, and DM automatically obtains the "max_allowed_packet" value to connect to TiDB.

Reviewer 2: ```suggestion
                                                # Since DM v2.0.0, this configuration item is deprecated, and DM automatically obtains the "max_allowed_packet" value from TiDB.
```

?

Author: Thanks for the comments~PTAL again@yikeke@csuzhangxc

 session:                                       # The session variables of TiDB, supported since v1.0.6. For details, go to `https://pingcap.com/docs/stable/system-variables`.

>     sql_mode: "ANSI_QUOTES,NO_ZERO_IN_DATE,NO_ZERO_DATE" # Since DM v2.0.0, if this item does not appear in the configuration file, DM will automatically fetch a proper value for "sql_mode" from the downstream TiDB. Manual configuration of this item has higher priority.

Reviewer: ```suggestion
    sql_mode: "ANSI_QUOTES,NO_ZERO_IN_DATE,NO_ZERO_DATE" # Since DM v2.0.0, if this item does not appear in the configuration file, DM automatically fetches a proper value for "sql_mode" from the downstream TiDB. Manual configuration of this item has a higher priority.
```

>     tidb_skip_utf8_check: 1                     # Since DM v2.0.0, if this item does not appear in the configuration file, DM will automatically fetch a proper value for "tidb_skip_utf8_check" from the downstream TiDB. Manual configuration of this item has higher priority.

Reviewer: ```suggestion
    tidb_skip_utf8_check: 1                     # Since DM v2.0.0, if this item does not appear in the configuration file, DM automatically fetches a proper value for "tidb_skip_utf8_check" from the downstream TiDB. Manual configuration of this item has a higher priority.
```
